install/migrations/update_9.4.x_to_9.5.0.php

```diff
@@ -576,8 +576,11 @@ function update94xto950()
 
     if (!$DB->fieldExists('glpi_entities', 'autopurge_delay')) {
         $migration->addField("glpi_entities", "autopurge_delay", "integer", [
-            'after'  => "autoclose_delay",
```

Comment:
I'm not sure this is a good ide to change so old migration files.

Reply:
They create inconsistant values but it seems that noone complains about it. I can revert these changes, or even move them into the main branch if you think it is too dangerous to change it in a bugfixes version.